src/components/Collection/Collection.tsx

```diff
@@ -0,0 +1,245 @@
+  const totalItems = numberOfItems;
+  const collectionId = CollectionUtils.getId(links);
+  const lastPageLink = links[links.length - 1].href;
+  const lastPage = parseInt(
+    lastPageLink.substring(lastPageLink.indexOf("=") + 1)
+  );
+  const firstElement = (currentPage - 1) * itemsPerPage + 1;
+  const lastElement =
+    currentCollectionQuery.page <= lastPage
+      ? currentPage * itemsPerPage
+      : totalItems;
+
+  const pageCount = Math.ceil(totalItems / currentCollectionQuery.perPage);
+
+  const getSortValue = () => {
+    const sortValue = Object.keys(collectionSortMap).find(
+      (key) => collectionSortMap[key].field === currentCollectionQuery.sort
+    );
+    return collectionSortMap[sortValue];
+  };
+
+  const sendCollectionQuery = async (collectionQuery: CollectionQuery) => {
+    router.push({
+      pathname: `/collection/${collectionQuery.identifier}`,
+      query: toLocationQuery(toApiCollectionQuery(collectionQuery)),
+    });
+  };
+
+  const onChangePerPage = (e) => {
+    e.preventDefault();
+    const newPage = 0;
+    const newPerPage = e.target.value;
+    if (newPerPage !== currentCollectionQuery.perPage) {
+      const newCollectionQuery: CollectionQuery = Object.assign(
+        {},
+        currentCollectionQuery,
+        {
+          page: newPage,
+          perPage: newPerPage,
+        }
+      );
+      setCollectionQuery(newCollectionQuery);
+      sendCollectionQuery(newCollectionQuery);
+    }
+  };
+
+  const onChangeSort = (e) => {
+    e.preventDefault();
+    if (
+      e.target.value !==
+      Object.keys(collectionSortMap).find(
+        (key) => collectionSortMap[key].field === currentCollectionQuery.sort
+      )
+    ) {
+      const newCollectionQuery: CollectionQuery = Object.assign(
+        {},
+        currentCollectionQuery,
+        {
+          sort: collectionSortMap[e.target.value].field,
+        }
+      );
+      setCollectionQuery(newCollectionQuery);
+      sendCollectionQuery(newCollectionQuery);
+    }
+  };
+
+  const onPageChange = (select: number) => {
+    const newCollectionQuery: CollectionQuery = Object.assign(
+      {},
+      currentCollectionQuery,
+      {
+        page: select,
+      }
+    );
+    setCollectionQuery(newCollectionQuery);
+    sendCollectionQuery(newCollectionQuery);
+  };
+
+  return (
+    <Template>
+      <TemplateBreakout>
+        <Breadcrumbs
+          breadcrumbsType="research"
+          breadcrumbsData={[
+            ...defaultBreadcrumbs,
+            {
+              url: `/collection/${collectionId}`,
+              text: truncateStringOnWhitespace(title, MAX_TITLE_LENGTH),
+            },
+          ]}
+        />
+        <Hero
+          backgroundColor="section.research.primary"
+          heroType="tertiary"
+          heading={
+            <Heading level="one" id="tertiary-hero">
+              <>
+                Digital Research Books <sup>Beta</sup>
+              </>
+            </Heading>
+          }
+        />
```

Comment:
Seems like this is a component that is duplicated in a couple of other places in the code. Let's pull it out into a separate component and re-use it in all the places it belongs.